packages/os/host-containers@.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=Host container: %i
 After=host-containerd.service
-BindsTo=host-containerd.service
+Wants=host-containerd.service
```

Comment:
`KillMode=mixed` in this unit file is misleading because there's only the one `host-ctr` process. Maybe a separate commit to clean that up, since it's our last use of that KillMode option.